pkg/compiler/test/policy_compiler_test.go

```diff
@@ -57,60 +70,83 @@ components:
 		swaggerContent string
 		result         string
 	}{
-		"blank swagger": {
-			swaggerContent: " ",
-			swaggerError:   errors.New("swagger is required for inferring types"),
-		},
-		"no swagger actions": {
-			swaggerContent: "openapi: \"3.0.0\"\ncomponents:\n  schemas:",
-			swaggerError:   errors.New("Swagger error: no actions are defined"),
-		},
-		"policies errors": {
-			packageName: "products.errors",
-			policyString: `
-				allow subject group everyone to inspect fake;
-				`,
-			compilerError: errors.New("expected next token to be TYPE_PATTERN, got IDENT instead\nexpected next token to be user or group, got IDENT instead"),
-		},
-		"products.inventory": {
+		// "blank swagger": {
+		// 	swaggerContent: " ",
```

Comment:
we should not be commenting out negative test cases

Reply:
Oops, sorry, I forgot to uncomment it after debugging